### DISCUSSION TITLE
S3 Generic Storage Backup

### DISCUSSION THREAD

**Discussion by yarlanda** [requarks/wiki#6214] [Error / Bug Report]
Hello,

Having the error message below when connecting Wiki to Minio S3 Storage
![2023-02-28_13h42_56](https://user-images.githubusercontent.com/85727403/221815365-6a4805cc-8391-49c4-a699-43b921b006fc.png)

I can confirm that the S3 is accessible and token has full rights. Below is my configuration
![2023-02-28_13h47_49](https://user-images.githubusercontent.com/85727403/221815850-9d2cd8ab-4288-4890-b55b-d2d47fc5daa5.png)

Wiki.js version 2.5.297 and is running on Official Docker Image
Thanks for the help

Cheers,
Yanick

**Comment by martivilella** [comment]
You need to enable "Force Path Style for S3 objects" for Minio. See here: https://github.com/requarks/wiki/issues/1472
